po/de.po

```diff
@@ -0,0 +1,90 @@
+# German translation for gnome-runcat-extension package.
+# Copyright (C) 2023 WebSnke
+# This file is distributed under the same license as the gnome-runcat-extension package.
+# WebSnke <websnke@tutanota.com>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-runcat-extension 21\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-13 16:30+0100\n"
+"PO-Revision-Date: 2023-03-13 16:30+0100\n"
+"Last-Translator: WebSnke\n"
+"Language-Team: none\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: src/panelMenuButton.js:95
+msgid "Open System Monitor"
+msgstr "Open System Monitor"
```

Comment:
Wrong translation: https://github.com/GNOME/gnome-system-monitor/blob/master/po/de.po#L39-L40

Reply:
I’ve added a few comments. The rest looks fine to me. :)

Reply:
> P.S. Let's wait a bit for a native speaker to review the PR. Thank you :-)

I am a native speaker...

Reply:
Translating it with "Freie Systemüberwachung" sounds very weird to me and I think that we should stick to the English description. That's why I left it as it was.

Reply:
I *think* `Open System Monitor` is the label of a button or a menu item that runs `gnome-system-monitor -r` when pressed. So maybe `Systemmonitor öffnen` could work here? I’m not a GNOME user, though, I don’t know how `gnome-system-monitor` is usually called – that term should probably be used here as well.

Reply:
Yep, "Systemmonitor öffnen" sounds good. I thought it meant "open" as in "open-source".

Reply:
> > P.S. Let's wait a bit for a native speaker to review the PR. Thank you :-)
> 
> I am a native speaker...

Oops. But actually, I mean I don't speak German, I asked a native speaker to review.